README.md

```diff
@@ -1,4 +1,68 @@
-# sales
-Public sale config
+# Tokensoft Sale Repository
+Public sale config for projects using Tokensoft tech to launch sales.
 
-Create a pull request for each sale using Tokensoft tech.
+## Intro
+To set up a new sale, follow these steps:
+1. Contact sales@tokensoft.io to engage Tokensoft for technology and compliance services.
+2. Create a new sale using https://app.tokensoft.io: this will provide the `saleId` value.
+3. Submit a pull request to this repository with the sale configuration.
+4. The sale will be accessible users once the pull request is approved by Tokensoft.
+5. To update this configuration once the sale is visible, create another pull request.
+
+## Setup
+On Github:
+* Sign in to github
+* Fork this repository
+
+On your machine:
+* Install git: https://github.com/git-guides/install-git
+* Clone the forked repository: `git clone git@github.com:<your github username>/sales.git`
+* Enter the repository: `cd sales`
+* Check out a new branch: `git checkout -b <the-name-of-my-sale>`
+* Install dependencies: `npm install`
+
+## Use
+On your machine:
+* Make a copy of the sale example: `cp sale.example.json ./sales/$YOUR_SALE_ID_HERE>.json`
```

Comment:
The config format is now a list of sales, so they can just add their config to the end of the config file instead of creating a new file. This can all be done directly from github.

Reply:
The repo lives here: https://github.com/tokensoft/sales